webapp-django/crashstats/cron/__init__.py

```diff
@@ -16,6 +16,14 @@
         'cmd': 'crontest',
         'frequency': '1d',
     },
+
+    {
+        # Audit hackers group
+        'cmd': 'auditgroups',
+        'cmd_args': ['--persist'],
+        'frequency': '7d',
+        'time': '05:00'
```

Comment:
This is pretty much all we need to run any Django command with arguments on a schedule. Neato!